libs/language-server/src/lib/constraint/regex-constraint-meta-inf.ts

```diff
@@ -16,5 +16,18 @@ export class RegexConstraintMetaInformation extends ConstraintMetaInformation {
       },
       ['text'],
     );
+    super.docs = {
+      description:
+        'Limits the values complying to a regex. Only values that comply with the regex are valid.',
+      examples: [
+        {
+          description:
+            'Text that complies with the IFOPT (Identification of Fixed Objects in Public Transport) format.',
+          code: `constraint IFOPT_Format oftype RegexConstraint {
```

Comment:
I found a specification for IFOPT values here: https://wiki.openstreetmap.org/wiki/DE:Key:ref:IFOPT#Syntax
Probably even more precisely in `DIN EN 28701:2012`.

The regex used here does not cover the structure of values correctly.

Reply:
Can you give me a hint what is correct?
If I see it correctly, there should be in this order, separated by `:`
* a sequence of 2 letters
* a sequence of x digits
* a sequence of x digits
* (optional) a sequence of  x digits
* (optional) a sequence of x digits


So the regex should be `/[a-z]{2}:\\d+:\\d+(:\\d+)?(:\\d+)?/`?

Reply:
Yes, I also think that this is right regex :+1: